src/sentry/wsgi.py

```diff
@@ -20,12 +20,15 @@
 # Run WSGI handler for the application
 application = WSGIHandler()
 
+
+warmup_host_name = settings.ALLOWED_HOSTS[0] if settings.ALLOWED_HOSTS[0] != "*" else "127.0.0.1"
```

Comment:
From https://docs.djangoproject.com/en/5.1/ref/settings/#allowed-hosts

> A value beginning with a period can be used as a subdomain wildcard: '.example.com' will match example.com, www.example.com, and any other subdomain of example.com.

And I see we have a few places where we set use these wildcards. Not sure if that'll cause issues or not.

Reply:
made the test for this use a wildcard ALLOWED_HOST in https://github.com/getsentry/sentry/commit/940f34f44afbeea9932279d38f0ff90a47407505